bits/src/main/scala/memeid/Bits.scala

```diff
@@ -75,7 +75,7 @@ private[memeid] object bits {
   }
 
   /* Convert to a Long from a byte seq. */
-  protected[memeid] def fromBytes(barr: Seq[Byte]): Long =
+  def fromBytes(barr: Seq[Byte]): Long =
```

Comment:
I marked this as `protected` so it can't be used from the outside, it makes assumptions about the input being a certain length.

Reply:
I removed it because the `bits` object is already marked as `private[memeid]`, isn't it already doing the same?